packages/discovery-react-components/rollup.config.js

```diff
@@ -61,7 +61,8 @@ export default {
       preferBuiltins: false
     }),
     commonjs({
-      exclude: [pdfWorkerRegex]
+      exclude: [pdfWorkerRegex],
+      namedExports: { 'react-pdf': ['Document', 'Page', 'pdfjs'] }
```

Comment:
This was needed because rollup was not properly bundling the package when doing a `yarn build`.  I mentioned this in our Slack DMs.  Apparently this is a known issue with `react-pdf` and how commonjs is used to bundle.   Supposedly this was fixed in a beta release, however I did test the latest beta release and was still having issues with rollup upon build.  Same issues that other users were experiencing.

Reply:
Maybe there's a better solution for handling this, if you have any suggestions.

Reply:
As a maintainer of React-PDF I'm also interested if there's something that can be done :) Try v5.0.0-beta.3 which ships with ES modules! It should be stable by now :)